packages/react-storage/src/components/StorageBrowser/createStorageBrowser/__tests__/createStorageBrowser.test-d.tsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable react/jsx-no-useless-fragment */
+import { expectAssignable, expectNotAssignable } from 'jest-tsd';
+import createStorageBrowser from '../createStorageBrowser';
+import { defaultActionConfigs } from '../../actions';
+import React from 'react';
+
+const { StorageBrowser, useAction, useView } = createStorageBrowser({
+  actions: {
+    default: {
+      ...defaultActionConfigs,
+    },
+    custom: {
+      myAction: {
+        handler: () => ({
+          result: Promise.resolve({ status: 'COMPLETE' }),
+        }),
+        viewName: 'MyActionView',
+        actionListItem: {
+          icon: 'info',
+          label: 'my action',
+        },
+      },
+    },
+  },
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  config: {} as any,
+});
+
+type StorageBrowserProps = Parameters<typeof StorageBrowser>[0];
+
+test('StorageBrowser component should support custom view and default view', () => {
```

Comment:
Does `jest-tsd` not allow usage of `describe`? Strong preference towards alignment with prior art for structuring unit test files

Reply:
Right, tsd file won't be actually executed, the tsd only checks the file against typescript compiler. So these descriptions will be not printed in tests. But write them in `describe` to group the tests better.